Countries Info/Managers/ApolloManager.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ApolloManager.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 26.11.2020.
+//
+
+import Foundation
+import Apollo
+
+class ApolloManager {
+    
+    static let shared = ApolloManager()
+    
+    private let url = URL(string: "https://countries-274616.ew.r.appspot.com/")
+    lazy var apollo = ApolloClient(url: url!)
```

Comment:
This property should be **private**, so we didn't have access to ApolloClient out of ApolloManager.